tests/integration/relationships/belongs-to-test.js

```diff
@@ -46,6 +47,11 @@ module('integration/relationship/belongs_to Belongs-To Relationships', {
       chapters: hasMany('chapters', { async: false, inverse: 'book' }),
     });
 
+    Book1 = DS.Model.extend({
+      name: attr('string'),
+      author: belongsTo('author', { async: false }),
```

Comment:
Given that in our test the `ModelData` for `book1` does not support relationships, we should consider removing `author` here. We should also add an assertion guarding for the case where `Author` defines  a relationship with an inverse to `book1` but `book1` does not support relationships. It is likely this would error out early due to invalid inverse configuration but being able to error out that "relationships aren't even supported" might be more useful in this scenario.